addons/website/static/tests/tours/o_editable.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import tour from "web_tour.tour";
+import wTourUtils from "website.tour_utils";
+
+tour.register("test_o_editable", {
```

Comment:
Confusing name as it makes me think of the "o_editable" class which is unrelated to the behavior of "o_not_editable". Maybe be more explicit with "test_drag_and_drop_on_non_editable" or something like that. We can always rename the test if we fill it with more stuff later.